cypress/e2e/common/cmsSteps.js

```diff
@@ -494,4 +494,16 @@ And('preview button was translated as {string}', (previewBtn) => {
 
 And('date published was translated as {string}', (datePublished) => {
     cy.get('ul.clearfix li strong').should('be.visible').and('include.text', datePublished);
+});
+
+And('the Card Title has a link {string} with href {string}', (linkText, linkHref) => {
+    cy.get(`a[href="${linkHref}"] h3:contains('${linkText}')`).should('be.visible');
+});
+
+And('feature card description reads {string}', (description) => {
+    cy.get(`p:contains('${description}')`).should('be.visible');
```

Comment:
could you make this locator more specific to point to feature card 
like .feature-card p:contains('${description}')

Reply:
done